legate/core/communicator.py

```diff
@@ -17,6 +17,8 @@
 from abc import ABC, abstractmethod, abstractproperty
 from typing import TYPE_CHECKING
 
+import numpy as native_np
```

Comment:
use `np`

Reply:
I just want to distinguish the numpy with cunumeric, so I rename it as native_np.

Reply:
we don't import cunumeric in legate core, so there's no need for such a distinction.